openfl/protocols/utils.py

```diff
@@ -44,6 +44,9 @@ def model_proto_to_bytes_and_metadata(model_proto):
                 f"Round numbers in model are inconsistent: {round_number} "
                 f"and {tensor_proto.round_number}"
             )
+    if round_number is None:
+        # For Federated Analytics, we are using a placeholder model with no tensors
+        round_number = 0
```

Comment:
This would likely not be needed if you implemented the suggestion above regarding `get_tensor_dict`...